src/Agent.Plugins/TestResultParser/PipelineConfig.cs

```diff
@@ -6,6 +6,7 @@ namespace Agent.Plugins.Log.TestResultParser.Plugin
     public class PipelineConfig : IPipelineConfig
     {
         public Guid Project { get; set; }
+
         public int BuildId { get; set; }
```

Comment:
>BuildId [](start = 19, length = 7)

we should call this PipelineRunId so that if needed for release this can be extended

Reply:
@nigurr to respond to this.

Reply:
Partly agrees with that. However current DevOps APIs differentiates BuildId/ReleaseId, would continue in that direction until we find a neutral place.